opsmngr/automation_config_test.go

```diff
@@ -190,6 +189,10 @@ func TestAutomation_GetConfig(t *testing.T) {
 		t.Fatalf("Automation.GetConfig returned error: %v", err)
 	}
 
+	if string(config.Raw()) != jsonBlob {
+		t.Fatalf("Raw returned config isn't equal origin")
+	}
```

Comment:
[q] isn't this always true?
in theory it should be just copying to `raw` here
https://github.com/mongodb/go-client-mongodb-ops-manager/blob/938536532fb6aab3921ceb98dafeb4bf0298fee0/opsmngr/opsmngr.go#L319-L321
Or is this just to test that the data is there?

Reply:
Yes, I want to validate we have data there. And if in future this will be changed I want to be sure that we really have origin data in the config struct